config/locales/en.yml

```diff
@@ -30,7 +30,34 @@
 # available at https://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
+  layouts:
+    application:
+      title: POD Aggregator
+  shared:
+    nav:
+      brand: Aggregator
+      providers: Providers
+      data: Data
+      activity: Activity
+      users: Users
+      view_org_page: '%{org} POD'
+      edit_profile: Edit profile
+      manage_users: Manage users
+      logout: Logout
+      login: Login
+    footer:
+      title: POD Aggregator
```

Comment:
i wondered about this. idk what the best practice is...do you try to avoid duplication here, or do you just sort of replicate all the strings in the view independently? would like to hear from more experienced rails folks @cbeer @corylown